build.gradle

```diff
@@ -280,7 +280,7 @@ subprojects {
         implementation 'org.slf4j:log4j-over-slf4j:1.7.30' // log4j1.2 bridge
 
         // Dependencies for logging to cloud storage, as well as the various clients used to do so.
-        implementation 'com.therealvan:appender-log4j2:3.1.2-SNAPSHOT'
+        implementation 'com.therealvan:appender-log4j2:3.2.0'
```

Comment:
Increased to a stable version so we don't have a weird "this is new" logging message.